driver-sync/src/test/functional/com/mongodb/client/AbstractAtlasSearchIndexManagementProseTest.java

```diff
@@ -84,7 +85,12 @@ private static boolean hasAtlasSearchIndexHelperEnabled() {
 
     @BeforeEach
     public void setUp() {
-        client = createMongoClient(getMongoClientSettings());
+        MongoClientSettings mongoClientSettings = MongoClientSettings.builder(getMongoClientSettings())
```

Comment:
There's a `getMongoClientSettingsBuilder` method that you can use instead.

Reply:
Done, thank you!